src/index.js

```diff
@@ -179,6 +179,20 @@ export default class ScrollBehavior {
     scrollTop(element, y);
   }
 
+  _defaultScrollTarget(hash) {
+    if (hash && hash !== '#') {
+      const id = hash.at(0) === '#' ? hash.substring(1) : hash;
```

Comment:
Good question! I actually have no idea where the definition of String#at is coming from. Chrome shows it as `function at() { [native code] }`

I don't know if there are browsers that don't start `hash` with a leading `#`, but  the `history` package supports it: https://github.com/ReactTraining/history/blob/d660be0fd29b63eb40398ba2a9d004b22284c965/modules/PathUtils.js#L46

It can happen as a result of a call like `history.push({pathname: 'faq', hash: 'Faq17'})`, though it seems not everything handles that gracefully.